### ISSUE TITLE
Plugins: Icons invisible when a large script is added on `head.end`

### ISSUE THREAD

**Issue by personalizedrefrigerator** [TypeStrong/typedoc#2589]
## Search terms

script, defer, svg, head.end, body.begin, plugin, theme

## Expected Behavior

The size of a script included in `head.end` should not affect whether icons are visible.

## Actual Behavior

Large scripts cause most icons (dropdown icons, etc.) to be invisible. Adding `defer` to the script helps, but the icons are still sometimes invisible.

![screenshot: Comparison between the expected and the actual behavior.](https://github.com/TypeStrong/typedoc/assets/46334387/368d8a1b-467b-4b9d-8f78-92c3f8ab34ee)

## Steps to reproduce the bug

A minimal [sample project with the issue can be found here](https://github.com/personalizedrefrigerator/bug-report/tree/typedoc/bug-include-script).

More specifically,
1. Clone https://github.com/personalizedrefrigerator/bug-report.git
2. Switch to the `typedoc/bug-include-script` branch.
3. Run `npm i`
4. Run `npm run build`
5. Run `npm run doc`
6. Start a server: `cd docs && python3 -m http.server`.
7. Open the documentation (tested in Firefox and Chromium on Ubuntu 24.04).

## Environment

-   Typedoc version: 0.25.13
-   TypeScript version: 5.4.5
-   Node.js version: v22.1.0
-   OS: Ubuntu 24.04

**Comment by personalizedrefrigerator** [comment]
To provide additional details, I have a custom TypeDoc theme that uses a `head.end` hook to add a script. For example,
```ts
renderer.hooks.on('head.end', (_event) => {
	return (
		// This happens with different libraries:
		//<script src="https://cdn.jsdelivr.net/npm/js-draw@1.20.3/dist/bundle.js" integrity="sha256-XJN7539P8FaFEZDXUEgMlp9ERBdhME2Gk7EoSZN2v3I=" crossOrigin="anonymous"></script>
		<script src="https://code.jquery.com/jquery-3.7.1.js" integrity="sha256-eKhayi8LEQwp4NKxN+CfCh+3qOVUtJn3QNZ0TciWLP4=" crossOrigin="anonymous"></script>
	);
});
```

For reference, `head.end`'s usage is [documented here](https://github.com/TypeStrong/typedoc/blob/33a34d6768510acd9ca143ec860dde87eadfc70e/internal-docs/custom-themes.md?plain=1#L71).

(Thank you for this excellent project!)

**Comment by Gerrit0** [comment]
Oops, that's busted... at least it's an easy fix!

**Comment by Gerrit0** [comment]
Fixed with 0.26, which is releasing 2024/06/21
